backend/stack.yaml

```diff
@@ -36,6 +36,8 @@ extra-deps:
   commit: 3dff2de34cce90c4d1dab68e03d30f8939fd52ca # not master
   subdirs:
     - servant-util
+- prometheus-metrics-ghc-1.0.1.1
+- wai-middleware-prometheus-1.0.0
```

Comment:
What’s the difference between `ai-middleware-prometheus` and `prometheus-wai-middleware`? I see the latter in Stackage.

Reply:
The difference is that `prometheus-wai-middleware` depend on [prometheus](http://hackage.haskell.org/package/prometheus) and `wai-middleware-prometheus` depend on [prometheus-client](http://hackage.haskell.org/package/prometheus-client).

Reply:
here is an example how to use `prometheus-wai-middleware`

```haskell
main :: IO ()
main = runRegistryT $ do
    ms <- applicationMetrics $ fromList [("app", "example_server")]
    let webserver = run 8080 . instrumentApplication ms $ \req respond ->
            respond (makeResponse req)
    _ <- liftIO $ async webserver
    serveMetricsT 8081 []
  where
    success = responseLBS status200 [] "ok"
    failure = responseLBS status404 [] mempty
    makeResponse req
        | pathInfo req == ["test"] = success
        | otherwise = failure
  ```

Reply:
I took a look at git and `prometheus` has last update two years ago

A think we can stay with `wai-middleware-prometheus` for MVP and further use our `servant-prometheus`.
(be aware, that it also uses `prometheus-client`)

Reply:
Taking all of the above into account, I think using `wai-middleware-prometheus` here is acceptable.